keyboards/handwired/unk/rev1/rules.mk

```diff
@@ -0,0 +1 @@
+OLED_ENABLE = yes
```

Comment:
If OLEDs are enabled at the keyboard level, there needs to be some OLED code at the keyboard level using the `oled_init_kb()` & `oled_task_kb()` functions.